src/styles/Book.scss

```diff
@@ -1,33 +1,132 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300;0,400;0,500;1,600&family=Roboto:wght@300;400;500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300;0,400;0,500;1,600&family=Montserrat&display=swap');
+
 $primaryColor: rgb(46, 164, 255);
 
 .container {
-  max-height: 67vh;
+  max-height: 72vh;
   overflow-y: auto;
   gap: 50px;
 
   .book {
     display: flex;
     justify-content: space-between;
     align-items: center;
-    padding: 20px 50px;
-    margin: 20px 1px;
+    padding: 10px 50px;
+    margin: 20px;
     background-color: rgb(255, 255, 255);
     font-size: 18px;
+    /* stylelint-disable */
+    box-shadow: rgba(0, 0, 0, 0.15) 2.4px 2.4px 3.2px;
+    /* stylelint-enable */
+
+    .left {
+      display: flex;
+      flex-direction: column;
+      justify-content: left;
+      width: 35%;
+    }
 
-    li {
+    .book-label {
+      height: 100px;
```

Comment:
- [ ] please, like mentioned in your project guidelines, Your final design should match Zeplin's design (this includes all elements + the same styling). Please kindly implement the different colors on the book title, author, genre, chapter, completed, comments etc. like in the original design.
- [ ] Also kindly ensure the book page title is aligned with the book's like in the original design.
- [ ] please, the user icon placeholder is aligned with the book cards on the right. Kindly implement that.
- [ ] please kindly add the extra margin under the form like in the design and ensure the add new book header and form should be aligned with the book cards. . nice work
- [ ] kindly add little extra margin left and right to the cards so the original design can be followed.
yours

![Screenshot (1139)_LI](https://github.com/mosidrum/bookstore/assets/65298048/981f0dd7-1604-460e-a5c4-2796830dafd4)
original design.
![Screenshot (1141)](https://github.com/mosidrum/bookstore/assets/65298048/7c25fbe7-4083-43ba-9d9d-b4467cd2cfb7)